src/Tribe/Views/V2/Widgets/Widget_List.php

```diff
@@ -80,6 +80,16 @@ class Widget_List extends Widget_Abstract {
 		],
 	];
 
+	/**
```

Comment:
:no_entry_sign: Doc comment for parameter "$arguments" missing

<sub>hash: c4c1045553e2dc32ecb1efbc265e194c</sub>